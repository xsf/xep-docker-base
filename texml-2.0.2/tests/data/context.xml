<?xml version="1.0"?>
<TeXML>
 <cmd name="enableregime">
  <opt>utf</opt>
 </cmd>
 <cmd name="setuppagenumbering">
  <opt>state=stop</opt>
 </cmd>
 <env name="text">
  <TeXML>Wie schön!</TeXML>
 </env>
 <env name="fake" start="xstart" stop="xstop">
  <opt>opt1</opt>
  <parm>par1</parm>
  <opt>opt2</opt>
  <parm>par2</parm>
  Wie schön!
 </env>
</TeXML>
