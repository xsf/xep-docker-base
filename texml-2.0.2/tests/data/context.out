\enableregime[utf] \setuppagenumbering[state=stop]
\starttext
Wie schön!
\stoptext
\xstartfake[opt1]{par1}[opt2]{par2}
Wie schön!
\xstopfake
